test/integration/test_b2_command_line.py

```diff
@@ -867,5 +870,21 @@ def main():
     print("ALL OK")
 
 
+# TODO: rewrite to multiple tests
```

Comment:
those are already split into like 5 different suites, can't we use them here?

Reply:
I think it's not that simple. I would keep it now as it is and try to do it better in separate PR

Reply:
right. Please file github issues for things we need to do in a separate PR, so that we don't miss anything

Reply:
Ok, I have a list. I will create an issue for that

Reply:
issue id?